### PR TITLE
[ch2849] Add checkbox to send notification to followers

### PR DESCRIPTION
**Pull request by cacahouwete**
- Add checkbox in committee form and validation

https://app.clubhouse.io/enmarche/story/2849/sur-la-page-d-envoi-de-message-pour-les-comit%C3%A9s-je-veux-rajouter-un-checkbox-pour-l-envoi-de-mails

### CODE REVIEW

File: src/Committee/Feed/CommitteeMessage.php [hunk 1]
```diff
@@ -17,7 +17,21 @@ class CommitteeMessage
      */
     private $content;
 
+    /**
+     * @Assert\Expression(
```
Comment (ottaviano) [inline src/Committee/Feed/CommitteeMessage.php]: can you apply `Assert\Expression` to the entire class?

File: src/Form/CommitteeFeedMessageType.php [hunk 2]
```diff
@@ -30,7 +30,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'purify_html' => true,
             ])
             ->add('published', CheckboxType::class, [
-                'label' => 'Rendre ce message visible sur la page du comité',
+                'label' => 'Publier sur la page du comité',
+                'attr' => ['class' => 'form__checkbox form__checkbox--large'],
+                'required' => false,
+            ])
+            ->add('send_notification', CheckboxType::class, [
```
Comment (ottaviano) [inline src/Form/CommitteeFeedMessageType.php]: have you tested the checked value of this field? 
I'm wondering for a mapping of this field `send_notification` and the class property `sendNotification`...

File: src/Committee/Feed/CommitteeMessage.php [hunk 3]
```diff
@@ -6,6 +6,12 @@
 use AppBundle\Entity\Committee;
 use Symfony\Component\Validator\Constraints as Assert;
 
+/**
+ * @Assert\Expression(
+ *     expression="this.isPublished() != false || this.isSendNotification() != false",
+ *     message="Vous devez selectionner au moins une des deux checkbox"
```
Comment (B-Galati) [inline src/Committee/Feed/CommitteeMessage.php]: I think we can improve this message and at least translate "checkbox"

Comment (B-Galati) [reply src/Committee/Feed/CommitteeMessage.php]: ~selectionner~
s**é**lectionner

You can double check with @guillaumedenhez or @marineb for the wording.

Comment (B-Galati) [reply src/Committee/Feed/CommitteeMessage.php]: Also :
case**s**

Comment (guillaumedenhez) [reply src/Committee/Feed/CommitteeMessage.php]: yes @cacahouwete good point from @B-Galati : 
- s**é**lectionner
- une des deux case**s** à cocher

**Pull request merged**